helix-view/src/editor.rs

```diff
@@ -918,6 +918,7 @@ pub struct Editor {
     pub auto_pairs: Option<AutoPairs>,
 
     pub idle_timer: Pin<Box<Sleep>>,
+    pub redraw_timer: Pin<Box<Sleep>>,
```

Comment:
```suggestion
    redraw_timer: Pin<Box<Sleep>>,
```

I think this one can be restricted to just this module, right? We wouldn't want anything else modifying the timer since it might mess up the debounce